deployment/docker/docker-compose-victorialogs.yml

```diff
@@ -72,26 +72,24 @@ services:
   # vmalert executes alerting and recording rules
   vmalert:
     container_name: vmalert
-    image: victoriametrics/vmalert:v1.105.0
+    image: victoriametrics/vmalert:heads-vmalert-support-vlog-ds-0-gdd5f198-scratch
```

Comment:
I think we should skip this until the actual release.

Reply:
Reverted to v1.105.0, and add vmauth as the datasource to proxy requests to both victoriametrics and victorialogs, so vmalert can execute metrics-related rules now and log-related rules will work from the next release.